### PR TITLE
Add yarn acceptance test

### PR DESCRIPTION
**Pull request by frbuceta**
See also https://github.com/strongloop/loopback-next/pull/5230#issuecomment-621934842

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [x] `npm test` passes on your machine
- New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- API Documentation in code was updated
- Documentation in [/docs/site](../tree/master/docs/site) was updated
- Affected artifact templates in `packages/cli` were updated
- Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: packages/cli/test/acceptance/app-run.acceptance.js [hunk 1]
```diff
@@ -55,7 +54,66 @@ describe('app-generator (SLOW)', function () {
...
+    // eslint-disable-next-line no-invalid-this
+    this.timeout(30 * 1000);
+
+    process.chdir(rootDir);
+    build.clean(['node', 'run-clean', appProps.outdir]);
+    process.chdir(process.cwd());
+  });
+});
+
+describe('app-generator with Yarn (SLOW)', function () {
```
Comment (raymondfeng) [inline packages/cli/test/acceptance/app-run.acceptance.js:77]: Is it going to fail if no `yarn` is installed on the test machine?

Comment (frbuceta) [reply packages/cli/test/acceptance/app-run.acceptance.js:77]: Yes, Lerna will not detect yarn.
Screenshot 2020-05-03 at 19 36 15

Comment (raymondfeng) [reply packages/cli/test/acceptance/app-run.acceptance.js:77]: Let's make the test conditional on the presence of `yarn` then.

Comment (frbuceta) [reply packages/cli/test/acceptance/app-run.acceptance.js]: Done

**Pull request merged**